test/utils.h

```diff
@@ -449,6 +452,10 @@ class NVFuserTest : public ::testing::Test {
                 << "'" << std::endl;
     }
   }
+
+#ifdef USE_DISTRIBUTED
+  static Communicator comm;
```

Comment:
I don't have anything specific, but thought it might be possible to prevent an error in one test affecting other following tests. Since we are not reestablishing MPI for each test, this won't make any different, though.

Reply:
> More generally than for the tests, we need to think about how/where to instantiate a communicator in nvFuser. Since we get WORLD_SIZE from the communicator, it might be used even at the scheduling step

I think the user should give us a communicator to use, which may be different `MPI_COMM_WORLD`. However, I don't think this matters at this moment.

Reply:
added as an item in #623